<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncDecDlg</class>
 <widget class="QDialog" name="EncDecDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Input data type</string>
   </property>
   <widget class="QRadioButton" name="mInputStringBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>String</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mInputHexBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mInputBase64Btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Base64</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="mInputText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Key and IV</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IV</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mIVTypeCombo"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="mIVText"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mKeyTypeCombo"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="mKeyText"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Method</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>376</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QComboBox" name="mMethodCombo"/>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="mAlgCombo"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mOutputTypeCombo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="mOutputText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EncDecDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EncDecDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
